training/training_code/train_ner.py

```diff
@@ -166,14 +169,38 @@ def _build_model(embedding_weights, char_bidirectional=False, concat_bidirection
 
 # === Build model ===
 
+num_classes = len(labels_arr_one_hot)
+f1 = F1Score(num_classes, average='micro')
+
 model = _build_model(embeddings)
 # Optimizer: Adam shows best results
 adam_op = Adam(lr=lr, decay=lr_decay)
-model.compile(optimizer=adam_op, loss='categorical_crossentropy', metrics=['accuracy'])
+model.compile(optimizer=adam_op, loss='categorical_crossentropy', metrics=['accuracy', f1])
+
+
+# Remove training output folder before saving the model files of the current run
+shutil.rmtree('training_output', ignore_errors=True)
+
+model_filename = "Epoch-{epoch:02d}"
+checkpoint_path = os.path.join('training_output/checkpoints/', model_filename)
+
+
+callbacks = ModelCheckpoint(filepath=checkpoint_path,
+                            period=1,
+                            verbose=1,
+                            save_weights_only=True,
+                            )
+
+'''
+# Load latest checkpoints and restart training
```

Comment:
Why is this commented out?

Reply:
I have commented this thinking we can use this part of the code only if checkpoints are available / want to start from the latest checkpoints.